problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,52 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob
+		    Use set notations, vectors, or linear combinations of vectors to describe the following sets.
+		    \begin{enumerate}
+		        \item The set of vectors with rational components in $\R^2$.
+		        \item The set of vectors with irrational components in $\R^2$.
+		        \item The set that contains the empty set.
+		        \item $\R^3$.
+		    \end{enumerate}
+    		
 		% Challenge (3 questions)
+		\prob 
+		% Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your answers.
+		    \begin{enumerate}
+		        \item The y-axis in $\R^3$.
+		        \begin{enumerate}
+		            \item It can be written as a set of linear combinations of vectors.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cup B$.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cap B$.
+		            \item For any vectors $\vec v \in \emptyset$, $\vec v \neq \vec 0$
+		            \item For some vectors $\vec v \in \emptyset$, $\vec v \neq \vec 0$.
+		        \end{enumerate}
+		        \item $\emptyset$
```

Comment:
I don't want to over-emphasize about the empty set. Perhaps the set of vectors whose coordinates are $\pm 3$?

Reply:
I used the empty set to distinguish from the previous set y-axis for the last two statements, as "for all x, P(x)" implies "for some x, P(x)" in general, but it is not the case for the empty set. It is a small detail about quantifiers. I got a feeling from marking last midterm's definition questions, some (or many) students do not understand quantifiers well. I can think about something better that doesn't use the empty set, or I can move these to the next question that asks for proof/counterexample?

Reply:
Go ahead and change it to my suggestion.